RELEASE/scripts/autoscend/auto_util.ash

```diff
@@ -4067,7 +4067,17 @@ boolean enforceMLInPreAdv()
 {
 	if((get_property("auto_MLSafetyLimit") != "") && (!contains_text(get_property("auto_maximize_current"), "ml")))
 	{
-		addToMaximize("ml " + get_property("auto_MLSafetyLimit").to_int() + "max");
+		if(get_property("auto_MLSafetyLimit").to_int() == 0)
+		{
+			// prevent all ML being equiped if limit is 0
+			addToMaximize("-1000ml");
```

Comment:
the problem with maximizer ML is that this would not prevent ML equip only, but also practically force equip anything with negative ml like red badge

Reply:
Yep you're right and that was my intent. If we are struggling, goal is to make fights easier. Including using negative ML.

In my Journeyman run I picked up the astral belt without thinking. Prior to this change the script equiped the belt even with safety limit set to 0 and I got beaten up a few times

Reply:
shouldn't it look for auto_MLSafetyLimit < 0 then if it's a temporary manual setting? otherwise there is no way user can set to not add any ML that doesn't also make them indiscriminately lose fight stats, equipment slots, mushroom pants etc would have to set at least +1 ML to avoid always forcing all available negative ML

not that the safety limit even works for equipment with current code, it only limits 1 out of 11 effective weight of ML without these changes #1097 and even with that fix the safety setting would not stop equipping astral belt for its other modifier, I think in the long term the only way autoscend can keep directly using maximizer gear is if the "max" weight argument is changed to work with negative weights on the mafia side

Reply:
Good point. I'll change it so that a setting of -1 causes all negative ML to be equipped. 

Agree that maximizer isn't working great for us in regards to capping ML and has been an issue for a while. Couldn't find a prior thread on mafia requesting this feature so made one.

https://kolmafia.us/threads/add-keyword-to-limit-a-modifier-in-maximizer.27601/